bcda/api.go

```diff
@@ -149,7 +149,10 @@ func bulkRequest(t string, w http.ResponseWriter, r *http.Request) {
 	}
 
 	acoID := ad.ACOID
-	userID := ad.UserID
+	user := models.User{}
+	// Arbitrarily use the first user in order to satisfy foreign key constraint "jobs_user_id_fkey" until user is removed from jobs table
+	db.First(&user)
```

Comment:
It's somewhat surprising that a possible mismatch between the job's acoID and user's acoID doesn't cause a problem, but I can confirm it doesn't, and this is ok.  One more step along the refactoring road!

Reply:
yep, i thought the same thing!